### ISSUE TITLE
[BUG] Nostr authentication on Firefox fails due to a recent nos2x-fox event signer update

### ISSUE THREAD

**Issue by ch0k1** [stackernews/stacker.news#1608]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Describe the bug

**Reproducibility rate:** Persistent

**Prerequisites:** 

1. Install [nos2x-fox](https://github.com/diegogurpegui/nos2x-fox) extension `v1.15.0` and configure it to use your private Nostr key for signing events;
2. The account should have Nostr account linked as a valid authentication method.

### Screenshots

<img width="750" alt="Screenshot 2024-11-19 at 1 36 53" src="https://github.com/user-attachments/assets/fe2a6bf2-4721-46fc-b1bf-579d971644f9">

### Steps To Reproduce

Start by opening a fresh SN home page (unauthenticated).

1. Click on `@anon`;
2. Click on `login` button;
3. Click on `Login with Nostr` button;
4. You get the following error:

```
error: nostr extension failed to sign event - nos2x-fox: Public key used doesn't match the active profile.
```

### Expected behavior

Should successfully authenticate to the SN account associated with the corresponding  Nostr pubkey configured in nos2x-fox  extension.

### Logs

<img width="663" alt="Screenshot 2024-11-19 at 2 14 18" src="https://github.com/user-attachments/assets/018ec4c8-6ea4-498a-bb31-d18dd84973d8">

### Device information

**Device:** MacBook Pro M3, Nov 2023
**OS:** MacOS Sequoia 15.1 (24B83)
**Browser:** Firefox 132.0.2 (aarch64)
**SN version:** [c31cf97](https://github.com/stackernews/stacker.news/commit/c31cf9)

### Additional context

***The issue is connected with the recent [nos2x-fox change](https://github.com/diegogurpegui/nos2x-fox/commit/3ddda7d24ff253d56391c9fc58315ae7c10acffa) which adds additional `activeKey` condition when attempting to sign Nostr events with an existing pubkey.***

**Comment by ekzyis** [comment]
Apparently v1.16.0 fixes this: https://github.com/diegogurpegui/nos2x-fox/issues/40#issuecomment-2484405019

It doesn't look like there is anything we can or should do.

**Comment by ekzyis** [comment]
Closing because no reply and afaict, this was fixed in v1.16.0 of `nos2x-fox`
